pkg/importer/dataStream.go

```diff
@@ -467,8 +483,18 @@ func closeReaders(readers []Reader) (rtnerr error) {
 	return rtnerr
 }
 
+func (d dataStream) isHTTPQcow2() bool {
+	// assuming len(d.Readers) == 3 is [http, mr, mr]
+	// should prob get rid of the redundant MultiReader
+	return (d.Url.Scheme == "http" || d.Url.Scheme == "https") && d.qemu && len(d.Readers) == 3
+}
+
 // Copy endpoint to dest based on passed-in reader.
 func (d dataStream) copy(dest string) error {
+	if d.isHTTPQcow2() {
+		glog.V(Vuser).Infoln("Doing streaming qcow2 to raw conversion")
```

Comment:
perhaps logging level here should be Vadmin? Just thinking that the end user does really care if we use qemu streaming, but the admin might due to SLA or perf guarantees.

Reply:
Makes sense